float32 ShoulderRot
float32 ShoulderPitch
float32 ElbowPitch
float32 WristPitch
float32 WristRot